bin/VMSlogger

```diff
@@ -116,6 +118,23 @@ parser.add_argument(
     dest="daemon",
     help="starts as daemon (fork to start process).",
 )
+parser.add_argument(
+    "--rotate",
+    action="store",
+    dest="rotate",
+    default=None,
+    type=parseDuration,
+    help="rotate on given interval",
+)
+parser.add_argument(
+    "--rotate-offset",
+    action="store",
+    dest="rotate_offset",
+    default=0,
+    type=parseDuration,
+    help="rotate at give offset from rotate interval (--rotate argument)",
```

Comment:
What would be the meaning of "rotate" if someone use the default value of 0? Never rotate? It might be good to improve the document here.

Reply:
That's for rotate_offset. 0 mens 0 offset from rotate, so rotate e.g. on midnight if someone specify 1D for rotate.